src/components/views/dialogs/ManualDeviceKeyVerificationDialog.tsx

```diff
@@ -107,14 +107,8 @@ export async function manuallyVerifyDevice(client: MatrixClient, deviceId: strin
             ),
         });
     } catch (e: any) {
-        let error = "Failed to verify device";
-        if (e.instanceof(UserFriendlyError)) {
-            error = (e as UserFriendlyError).translatedMessage;
-        } else if (e.instanceof(Error)) {
-            error = (e as Error).message;
-        }
-
         // Display an error
+        const error = e instanceof UserFriendlyError ? e.translatedMessage : e.toString();
```

Comment:
worth noting that `(new Error("foo")).toString()` produces `Error: foo`, whereas using `.message` just gives you `foo`.

I guess that's fine.